content/en/agent/kubernetes/daemonset_setup.md

```diff
@@ -201,15 +201,39 @@ To enable [Log collection][8] with your DaemonSet:
       (...)
     ```
 
-Learn more about this in [the Docker log collection documentation][9].
+Learn more about this in [the Docker log collection documentation][10].
 
-### Trace Collection
+### APM and Distributed Tracing
 
-To enable trace collection with your DaemonSet:
+To enable APM, set the `DD_APM_NON_LOCAL_TRAFFIC` variable to true in your *env* section:
 
-1. Set the Node IP and port as environment variables for your application containers:
+```
+(...)
+      env:
+        (...)
+        - name: DD_APM_NON_LOCAL_TRAFFIC
+          value: "true"
+(...)
+```
 
-    ```
+This is needed for the container to allow incoming data from port 8126.
+
+Then, forward the port of the Agent to the host. 
+
+```
+(...)
+      ports:
+        (...)
+        - containerPort: 8126
+          hostPort: 8126
+          name: traceport
+          protocol: TCP
+(...)
+```
+
+Use the downward API to pull the host IP; the application container needs an environment variable that points to `status.hostIP`. You can name it `DD_AGENT_HOST`:
+
+```
```

Comment:
Indentation needed here so the markdown code block doesn't break.

Reply:
Potential wording fix here:

Use Kubernetes' downward API to pull the host Node's IP with `status.hostIP`.  The Datadog container Agent expects this to be named `DD_AGENT_HOST`:

Note the `can name it`  isn't really appropriate. It must be named that to work.